app/views/idv/phone_errors/failure.html.erb

```diff
@@ -39,3 +43,5 @@
         </div>
       <% end %>
     <% end %>
+    <%= link_to(t('links.cancel'), return_to_sp_cancel_url) %>
```

Comment:
```suggestion
    <%= render PageFooterComponent.new do %>
      <%= link_to t('links.cancel'), idv_cancel_path %>
    <% end %>
```

Reply:
Nope. The ticket specifies that that link should return to the SP, not just cancel IdV. Seems weird to me too, but.